kb-importer/pom.xml

```diff
@@ -1,47 +1,60 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-
     This file is part of Eclipse Steady.
-
     Licensed under the Apache License, Version 2.0 (the "License");
     you may not use this file except in compliance with the License.
     You may obtain a copy of the License at
-
         http://www.apache.org/licenses/LICENSE-2.0
-
     Unless required by applicable law or agreed to in writing, software
     distributed under the License is distributed on an "AS IS" BASIS,
     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     See the License for the specific language governing permissions and
     limitations under the License.
-
     SPDX-License-Identifier: Apache-2.0
     SPDX-FileCopyrightText: Copyright (c) 2018-2020 SAP SE or an SAP affiliate company and Eclipse Steady contributors
-
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
-		<artifactId>root</artifactId>
-		<groupId>org.eclipse.steady</groupId>
-		<version>3.2.5-SNAPSHOT</version>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.6.3</version>
 	</parent>
-
+	<groupId>org.eclipse.steady</groupId>
 	<artifactId>kb-importer</artifactId>
-
+	<version>3.2.3-SNAPSHOT</version>
```

Comment:
That should be 3.2.5-SNAPSHOT...

Reply:
Please compare with `rest-lib-utils/pom.xml` and copy/adapt relevant tags, e.g., the description, profiles, or properties.